### PR TITLE
Add release notes for Omni Core 0.0.11

### PR DESCRIPTION
**Pull request by dexX7**
This pull request adds the release notes for Omni Core 0.0.11.

See the preview here:

https://github.com/dexX7/bitcoin/blob/c367a0ba6a179cd369539e9168fa68e86b9eb6d8/src/omnicore/doc/release-notes.md

Suggestions for the items are very welcomed! :)

### CODE REVIEW

File: src/omnicore/doc/release-notes.md [hunk 1]
```diff
@@ -64,498 +49,132 @@ Omni Core is based on Bitcoin Core 0.10.4 and can be used as replacement for Bit
...
-
-Various performance improvements
---------------------------------
+Omni Core 0.0.11 adds support for payload creation via the RPC interface.
 
-Omni Core was improved on several levels to increase performance and responsiveness.
+The calls are similar to the send transactions (e.g. `omni_send`), without the requirement for an address or any of the balance checks.
 
-Previously RPC requests were blocking, creating a bottleneck for the JSON-RPC API and the whole application. Due to much finer and more targeted locking of critical sections, threads are now only blocked when they need to be, which allows concurrent requests without immediately creating a queue. The number of threads designated to handle RPC requests can be configured with the option `-rpcthreads=<n threads>`, which is set to `4` threads as per default.
+This allows an integrator building transactions via the [raw transactions interface](https://github.com/OmniLayer/omnicore/blob/omnicore-0.0.10/src/omnicore/doc/rpc-api.md#raw-transactions).
```
Comment (zathras-crypto) [inline src/omnicore/doc/release-notes.md]: This sentence feels incomplete.

File: src/omnicore/doc/release-notes.md [hunk 2]
```diff
@@ -64,498 +49,132 @@ Omni Core is based on Bitcoin Core 0.10.4 and can be used as replacement for Bit
...
-The configuration option `-overrideforcedshutdown` can be used to prevent Omni Core from shutting down in case of a checkpoint mismatch. Using the latter is generally not recommended, as this exposes the user to data, which isn't considered as valid by other participants.
+The field `"positioninblock"` was added to RPCs retrieving or listing Omni transactions to provide visibility into the order of an Omni transaction within a block.
 
-Disable-wallet mode
--------------------
+Increased OP_RETURN payload size to 80 bytes
+--------------------------------------------
 
-Since this version Omni Core can be build and used without wallet support.
+The maximum size for OP_RETURN outputs used to be 80 bytes, then got changed to 40 bytes to be on the safe side, and then was increased once again.
```
Comment (zathras-crypto) [inline src/omnicore/doc/release-notes.md]: Did we ever release a 80 byte OP_RETURN initially with class C or did we activate at 40?  If so perhaps it's just necessary to say it's been increased?

**Comment by zathras-crypto** [comment]
Looks awesome thanks dude, couple of minor nits but feel free to go ahead whenever you're ready :)

File: src/omnicore/doc/release-notes.md (hunk 2 above)
Comment (dexX7) [inline src/omnicore/doc/release-notes.md]: Good point. It sounds a bit strange to me too.

**Comment by dexX7** [comment]
Awesome, thanks!

**Pull request merged**